src/main/java/org/jabref/cli/ArgumentProcessor.java

```
@@ -289,6 +294,45 @@ public void processArguments() {
         if (!cli.isBlank() && !loaded.isEmpty()) {
             uiCommands.add(new UiCommand.OpenDatabases(loaded));
         }
+
+        if (cli.isCheckConsistency()) {
+            checkConsistency();
+        }
+    }
+
+    private void checkConsistency() {
+        String fileName = cli.getCheckConsistency();
+        String outputFormat = cli.getCheckConsistencyOutputFormat().toUpperCase();
+
+        if (fileName == null) {
+            System.out.println(Localization.lang("No file specified for consistency check."));
+            return;
+        }
```

Move this up to after `filename` -> fail fast